bin/bib_ids.txt

```diff
@@ -1,4872 +0,0 @@
-10080579
```

Comment:
I don't think we should delete all of the old tooling for retrieving MARC records. This might change, but I expect we will still want to get MARC from NGA, load it into Alma, and update what's in this repo.

Reply:
I've added them back to the repo